src/helpers/loadOrCreatePrizePool.ts

```diff
@@ -45,6 +50,11 @@ export function loadOrCreatePrizePool(
 
     _prizePool.reserveFeeControlledToken = boundPrizePool.reserveFeeControlledToken()
 
+    if (prizePoolType === 'Compound') {
+      const boundCompoundPrizePool = CompoundPrizePoolContract.bind(prizePool)
+      _prizePool.yieldToken = boundCompoundPrizePool.cToken()
```

Comment:
Two questions:

- What happened to having a separate CompoundPrizePool entity?  
- Why is this being renamed from cToken to yieldToken?

Reply:
Will be addressed in ticket: feature/pool-545-link-prizepool-to-compoundprizepool